spec/controllers/families_controller_spec.rb

```diff
@@ -31,7 +31,10 @@
       it "destroys the family" do
         family = create(:family)
 
-        expect { delete :destroy, params: { id: family.id } }.to change(Family, :count).by(-1)
+        expect { delete :destroy, params: { id: family.id } }
+          .to change(
```

Comment:
The `.` should be in the end of line 34. But I would write it:
```
expect do
  delete :destroy, params: { id: family.id }
end.to change(Family, :count).by(-1)
```

Reply:
25ddc19